testing/SPEC-CONFORMANCE.md

```diff
@@ -130,10 +129,9 @@ The conformance requirements for EBU-TT Part 3 derive from the specification its
 |R117|2.3.4.2|A Retiming Delay node shall not update the value of `ebuttm:authoringDelay`.|`bdd/features/timing/retimingDelayNode.feature`|
 |R118|2.2|Two documents are considered identical if the result of the fn:deep-equal function [XFUNC] is true when both documents are provided as operands.|NOTE: Due to partial support for XQuery functions in the Python libraries that we use, this requirement is not implemented fully in this toolkit. Equality testing in other implementation may produce slightly different results.|
 |R119|2.3.4.1|A Buffer Delay node is a passive node. Therefore the output documents shall be identical to the input documents.|`bdd/features/nodes/passive\_nodes\_shall\_not\_modify\_document.feature`|
-||| **March 2017**||
-|R120|2.4.1|The Handover Manager shall include within each output document the metadata attribute `authorsGroupSelectedSequenceIdentifier` set to the value of the source sequence's identifier for that document.||
-|R121|2.4.1|The Handover Manager shall not emit any documents derived from documents that do not contain both the parameters `ebuttp:authorsGroupIdentifier` and `ebuttp:authorsGroupControlToken`.||
-|R122|3.2.2.1|The value of `ebuttp:authorsGroupControlToken` shall be a positive integer.||
-|R123|3.2.2.1|The value of `ebuttp:sequenceNumber` shall be a positive integer.||
-|R124|3.2.2.1|The value of `ebuttp:sequenceIdentifier` shall be a string of length >= 1.|`bdd/features/validation/sequence\_id\_num.feature` `(In)valid Sequence head attributes`|
-|R125|3.2.2.1|The value of `ebuttp:authorsGroupIdentifier` shall be a string of length >= 1.||
+||| **Tech 3370 v1.0**||
+|R120|2.4.2|The Handover Manager shall maintain all of the following variables: a token equal to the value of the `ebuttp:authorsGroupControlToken` of the most recently emitted document, or a null value if no document has been emitted; a record of the sequence identifier of the source sequence used to generate the most recently emitted document, known as the selected sequence, initially a null value until a sequence is selected; a record of the configured authors group identifier; an output sequence identifier.|Tested implicitly as part of the algorithm in `bdd/features/handover/handover_algorithm.feature`|
```

Comment:
We never had shall requirements for each step of the algorithm. This is what I was getting at here: https://ebu-tt-lit.slack.com/archives/D1HQ8HNKD/p1493386349784949